services/QuillLMS/spec/services/diagnostic_insights/diagnostics_assigned_by_classroom_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+module DiagnosticInsights
+  describe DiagnosticsAssignedByClassroom do
+    subject { described_class.run(classroom_id) }
+
+    let(:classroom) { create(:classroom) }
+    let(:classroom_id) { classroom.id }
+    let(:pre_diagnostic) { create(:pre_diagnostic_activity) }
+    let(:user) { create(:student) }
+    let(:assigned_student_ids) { [user.id] }
+    let(:unit) { create(:unit, activities: [pre_diagnostic]) }
+    let(:classroom_unit) { create(:classroom_unit, classroom:, unit:, assigned_student_ids:) }
+    let(:student_learning_sequence) { create(:student_learning_sequence, user:, initial_activity_id: pre_diagnostic.id, initial_classroom_unit: classroom_unit) }
+    let!(:student_learning_sequence_activity) { create(:student_learning_sequence_activity, student_learning_sequence:, activity: pre_diagnostic) }
+    let(:diagnostic_key) { described_class::STARTER_DIAGNOSTIC }
+    let(:expected_payload) do
+      [{
+        activity_id: student_learning_sequence_activity.activity_id,
+        completed_at: student_learning_sequence_activity.completed_at,
+        created_at: student_learning_sequence_activity.created_at,
+        user_id: student_learning_sequence_activity.student_learning_sequence.user_id
+      }]
+    end
+
+    before do
+      stub_const('Activity::STARTER_PRE_DIAGNOSTIC_IDS', [pre_diagnostic.id])
+    end
+
+    it { expect(subject).to be }
```

Comment:
These look like unfinished tests.

Reply:
As I understand it, and my testing suggests this is correct, `expect(...).to be` with no final argument tests the truthiness of the expectation.  I'm using it as `expect(...).not_to be(nil)`.  Do you think the more verbose option is clearer because it's an easy change to make.